### ISSUE TITLE
Type limited to 4 images

### ISSUE THREAD

**Issue by Jeremy-M-Int** [INTERPOL-Innovation-Centre/ANSI-NIST-XML-ITL-Implementation#7]
The current wording in the standard specifies this:
"At this time, the present standard allows the transmission of facial images (up to 4) only in this record."

We are aiming at INTERPOL to allow up to 10 images.

It would makes sense tp update the standard to reflect this unless there is good reason to limit to 4.

**Comment by tsaracouvert** [comment]
Agreed to increase the max number of images to 10 facial images. SMT are not allowed for now in the standard.
